Resources/Prototypes/Body/Parts/animal.yml

```diff
@@ -0,0 +1,187 @@
+
+- type: entity
+  id: TorsoAnimal
+  name: "animal torso"
+  parent: PartAnimal
+  abstract: true
+  components:
+  - type: BodyPart
+    partType: Torso
+    size: 7
+    compatibility: Biological
+    mechanisms:
+    - OrganAnimalLungs
+    - OrganAnimalStomach
+  - type: Damageable
+    damageContainer: Biological
+
+- type: entity
+  id: BaseAnimalOrgan
+  parent: BaseItem
+  abstract: true
+  components:
+  - type: Mechanism
+
+- type: entity
+  id: OrganAnimalLungs
+  parent: BaseAnimalOrgan
+  name: lungs
+  abstract: true
+  components:
+  - type: Mechanism
+    size: 1
+    compatibility: Biological
+    behaviors:
+    - !type:LungBehavior {}
+
+- type: entity
+  id: OrganAnimalStomach
+  parent: BaseAnimalOrgan
+  name: stomach
+  abstract: true
+  components:
+  - type: Mechanism
+    size: 1
+    compatibility: Biological
+    behaviors:
+    - !type:StomachBehavior
+      max_volume: 100
+      digestionDelay: 20
+  - type: SolutionContainerManager
+    solutions:
+      stomach:
+        maxVol: 100
+  - type: Metabolizer
+    metabolisms:
+      Flour:
+        effects:
+        - !type:SatiateHunger
+      JuiceApple:
+        effects:
+        - !type:SatiateThirst
+      JuiceBerry:
+        effects:
+        - !type:SatiateThirst
+      JuiceBanana:
+        effects:
+        - !type:SatiateThirst
+      JuiceCarrot:
+        effects:
+        - !type:SatiateThirst
+      JuiceLime:
+        effects:
+        - !type:SatiateThirst
+      JuiceLemon:
+        effects:
+        - !type:SatiateThirst
+      JuiceGrape:
+        effects:
+        - !type:SatiateThirst
+      JuiceOrange:
+        effects:
+        - !type:SatiateThirst
+      JuiceTomato:
+        effects:
+        - !type:SatiateThirst
+      JuiceBerryPoison:
+        effects:
+        - !type:SatiateThirst
+        - !type:HealthChange
+          damage:
+            types:
+              Poison: 1
+      JuiceWatermelon:
+        effects:
+        - !type:SatiateThirst
+      JuicePineapple:
+        effects:
+        - !type:SatiateThirst
+      Nutriment:
+        effects:
+        - !type:SatiateHunger
+      Water:
+        effects:
+        - !type:SatiateThirst
+          hydrationFactor: 2
+      Coffee:
+        effects:
+        - !type:SatiateThirst
+      Tea:
+        effects:
+        - !type:SatiateThirst
+      Milk:
+        effects:
+        - !type:SatiateThirst
+      MilkSpoiled:
+        effects:
+        - !type:SatiateThirst
+          hydrationFactor: -2
+      MilkSoy:
+        effects:
+        - !type:SatiateThirst
+          hydrationFactor: 2
+      MilkOat:
+        effects:
+        - !type:SatiateThirst
+          hydrationFactor: 2
+      Cola:
+        effects:
+        - !type:SatiateThirst
+          hydrationFactor: 0.5
+      FourteenLoko:
+        effects:
+        - !type:SatiateThirst
+          hydrationFactor: 2
+        - !type:HealthChange
+          damage:
+            types:
+              Poison: 1
```

Comment:
this is REALLY ass but i'm okay with it for now since I'm currently doing a metabolism refactor that'll make this obsolete